lib/stdlogger.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const EventEmitter = require('events');
+const Hoek = require('@hapi/hoek');
+
+const internals = {};
+
+class Stdlogger extends EventEmitter {
+    constructor(stdout, stderr) {
+
+        super();
+        this.stdout = stdout;
+        this.stderr = stderr;
+    }
+}
+
+[
+    'emergency',
+    'alert',
+    'critical',
+    'error'
+].forEach((level) => {
+
+    Stdlogger.prototype[level] = function (msg, data, additionalFields) {
+
+        const stringified = internals.stringify(level, msg, data, additionalFields);
+        this.stderr.write(`${stringified}\n`);
```

Comment:
Just a note, writing to stderr will be a small incompatibility with pino transports— but it's easy enough to account for for e.g. with `2>&1`.

Reply:
This was intentional... going against historical norms seemed odd.